lib/email/bounces.js

```diff
@@ -52,56 +45,66 @@ module.exports = function (log, error) {
 
     function handleBounce(message) {
       var recipients = []
-      if (message.bounce && message.bounce.bounceType === 'Permanent') {
```

Comment:
> would we want to delete unverified accounts because of soft bounces?

I think this is probably reasonable, but would prefer to tackle it as a separate issue rather than conflating with this feature.